### PR TITLE
possible to turn off human readbal and push/pull classes

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: src/scss/_main.scss [hunk 1]
```diff
@@ -606,8 +606,13 @@
 ///
 /// @param {String} $layout-name - One of $o-grid-layouts
 @mixin _oGridModifiersForLayout($layout-name) {
-	@include _oGridHumanFriendlyKeywords($layout-name);
-	@include _oGridShuffleColumns($layout-name);
+	@if $o-grid-human-friendly-selectors == true {
+		@include _oGridHumanFriendlyKeywords($layout-name);
+	}
+
+	@if $o-grid-offset-selectors == true {
```
Comment (i-like-robots) [inline src/scss/_main.scss]: maybe this should be called `$o-grid-shuffle-columns`  as well?

Comment (onishiweb) [inline src/scss/_main.scss]: Agreed, we should probably aim to stick with current naming schemes to prevent confusion

**Comment by onishiweb** [comment]
Would like to discuss this in next week's Origami Open Meeting as it closely relates to https://github.com/Financial-Times/ft-origami/issues/483

**Comment by AlbertoElias** [comment]
If silent mode is on, these mixins won't be output anyway, right?

**Comment by wheresrhys** [comment]
we want non-silent mode only for the core functionality

**Comment by AlbertoElias** [comment]
I guess this is related to all the other silent mode conversations, so we
can talk about it on Thursday

On Fri, 27 May 2016 at 15:58 Rhys Evans <notifications@github.com> wrote:

> we want non-silent mode only for the core functionality
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Financial-Times/o-grid/pull/132#issuecomment-222168852>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA3u3YDVdq87fdEMq_kDBrmcLTz8yIW-ks5qFwZ7gaJpZM4IgLzb>
> .
>

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*

**Pull request closed without merging**